import-automation/executor/gke/README.md

```diff
@@ -26,8 +26,14 @@ Follow
 
 ## (One Time) Setup GKE
 
-1. Update OAUTH_CLIENT_ID and OAUTH_CLIENT_SECRET in "gke/configure_gke.sh".
-2. Run `./gke/configure_gke.sh`.
+1. Set the PROJECT_ID, OAUTH_CLIENT_ID and OAUTH_CLIENT_SECRET environment variables in "gke/configure_gke.sh", e.g.
+```
+export PROJECT_ID=<GCP_PROJECT>
+export PROJECT_ID=<OAUTH_CLIENT_ID>
+export PROJECT_ID=<OAUTH_CLIENT_SECRET>
```

Comment:
update variable name in line 32 and 33

Reply:
Thanks! Took me a while to spot the issue :)